projects/control-service/cicd/start_heartbeat.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+# Copyright 2021 VMware, Inc.
+# SPDX-License-Identifier: Apache-2.0
+
+pip install -U $VDK_HEARTBEAT_VDK_DISTRIBUTION_NAME --extra-index-url $VDK_HEARTBEAT_PIP_EXTRA_INDEX_URL
```

Comment:
Set default . 

One of our goals is to make everything we do easy to get started with. That means no required configurations as much as possible in this case. We should aim for every configuration value we expose should have some sensible default -> less things for users to have to configure at first -> This way we reduce start up time for new users -> happier users. 


```
VDK_HEARTBEAT_VDK_DISTRIBUTION_NAME=${VDK_HEARTBEAT_VDK_DISTRIBUTION_NAME:-'quickstart-vdk'}
VDK_HEARTBEAT_PIP_EXTRA_INDEX_URL=${VDK_HEARTBEAT_PIP_EXTRA_INDEX_URL:-'https://pypi.org/simple'}

Reply:
Done, thanks